signer/src/crypto/secret_uri.rs

```diff
@@ -115,14 +117,14 @@ impl std::str::FromStr for SecretUri {
 }
 
 /// This is returned if `FromStr` cannot parse a string into a `SecretUri`.
-#[derive(Debug, Copy, Clone, PartialEq, thiserror::Error)]
+#[derive(Debug, Copy, Clone, PartialEq, Display)]
```

Comment:
impl std::error::Error for this one? I saw that James commented on it but I can't find the impl :)